components/picker/index.css

```diff
@@ -214,7 +212,7 @@ governing permissions and limitations under the License.
   /* Start with text-only padding */
   padding-inline: var(--mod-picker-spacing-edge-to-text, var(--spectrum-picker-spacing-edge-to-text));
 
```

Comment:
I believe picker needs

`position: relative;`

so focus indicator has something to align off of